addons/mail/models/mail_mail.py

```diff
@@ -697,7 +697,13 @@ def _send(self, auto_commit=False, raise_exception=False, smtp_session=None, ali
                             raise
                 if res:  # mail has been sent at least once, no major exception occurred
                     mail.write({'state': 'sent', 'message_id': res, 'failure_reason': False})
-                    _logger.info('Mail with ID %r and Message-Id %r successfully sent', mail.id, mail.message_id)
+                    _logger.info(
+                        "Mail with ID %r and Message-Id %r from %r to (redacted) %r successfully sent",
+                        mail.id,
+                        mail.message_id,
+                        tools.email_normalize(msg['from']),
+                        tools.mail.email_anonymize(tools.email_normalize(msg['to']))
```

Comment:
Would use email_normalize_all, as you don't always have a single to email

Reply:
```py
>>> tools.email_normalize('a@b.c,d@e.f')
False
>>> tools.email_anonymize(False)
False
```
Not great (it prints "False" which is useless), not terrible (it doesn't crash).

Would the following be an acceptable solution?
```suggestion
                        ','.join([
                            tools.mail.email_anonymize(to)
                            for to in tools.email_normalize_all(msg['to'])
                        ])
```